etc/picongpu/bash-devServer-hzdr/cpu_picongpu.profile.example

```diff
@@ -30,10 +30,10 @@ spack load cmake@3.23.1 %gcc@10.2.0
 spack load openpmd-api@0.14.4 %gcc@10.2.0 \
     ^adios2@2.8.0 \
     ^hdf5@1.12.1 \
-    ^openmpi@4.1.3 +atomics \
+    ^openmpi@4.1.3 +atomics ~cuda\
```

Comment:
I think this line is wrong. Why should we use an MPI without CUDA support? IS is very easy making problems.

Reply:
because this is cpu-only

Reply:
there is a separate profile for gpu which explicitly uses cuda

Reply:
https://github.com/ComputationalRadiationPhysics/picongpu/blob/a17e2aa829e3f36c211959bf94f5d7121a3fba65/etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example#L33

Reply:
also this is intended as a minimal profile, therefore if we do not need it, it is not there